### DISCUSSION TITLE
Compatibility with ags v2?

### DISCUSSION THREAD

**Discussion by aahsnr-configs** [Jas-SinghFSU/HyprPanel#310] [Q&A] [answered]
Would your config be compatible with v2 of ags?

For manually install ags v1 I get an typescript error in debian based distros but v2 installs fine.

**Comment by Jas-SinghFSU** [comment]
When v2 releases I'll 100% be moving to it. But I don't think it's completed yet.

**Comment by aahsnr-configs** [comment]
Yeah the wiki says soon but I will still try out your config with v2

-------- Original Message --------

**Comment by Jas-SinghFSU** [comment]
@zielOS What error do you get btw?

**Comment by aahsnr-configs** [comment]
Hi @Jas-SinghFSU I managed to fix the issue of installing ags in debian based distros on my own. The 1st problem was that node-typescript was older than 5.0 and ags requires a much higher version, thats why it gave an error with ); at the end of each line in the error code. I managed to install the latest version with npm but had to add both npm-global/bin and npm-global/lib to path to install properly. Without adding lib the installation failed to locate tsc.js.

On an off-topic subject, do you plan to add an application launcher to your HyprPanel. I noticed the dock at the middle thats why I asked. It's technically not a panel anymore, more like a shell since I can do anything aylur's dotfiles could do except the application launcher. Good job btw....it's a much more complete panel on its own right

**Reply by Jas-SinghFSU** [reply, marked as answer]
Nice, glad you've got it working. As for the Applications Launcher, I'm not certain if I want to or not. My concern is that AGS requires a restart everything time an application is added (if I'm not mistaken) and Wofi already has so much out of the box that's available. 

I'd need to think about it.

**Reply by anotherhadi** [reply]
> As for the Applications Launcher, I'm not certain if I want to or not. My concern is that AGS requires a restart everything time an application is added (if I'm not mistaken) and Wofi already has so much out of the box that's available.

Maybe just create a wofi/rofi theme based on the hyprpanel colorscheme?

**Reply by aahsnr-configs** [reply]
> Nice, glad you've got it working. As for the Applications Launcher, I'm not certain if I want to or not. My concern is that AGS requires a restart everything time an application is added (if I'm not mistaken) and Wofi already has so much out of the box that's available.
> 
> I'd need to think about it.

Yeah it's totally upto you. For some reason, as I don't know the coding language but Aylur's own dotfiles manages to create a launcher and when I was using it in non-debian based distro ags didn't restart when I launched an application.
